### ISSUE TITLE
OAI functionality

### ISSUE THREAD

**Issue by datadavev** [CDLUC3/ezid#244]
Various OAI API methods are failing because they reference Identifier instead of SearchIdentifier models.

**Comment by datadavev** [comment]
The list identifiers OAI request is timing out. Seems to need a new index. Tracking it down.

**Comment by datadavev** [comment]
It was necessary to add a new index:

```
create index ezidapp_searchidentifier_updaTime_idx on ezidapp_searchIdentifier (updateTime);
```
The index took approximately 50 minutes to build, and did not appear to disrupt normal operations. After index creation, the time out issues are resolved, however see #268
